src/main/scala/org/zalando/spearheads/innkeeper/Rejections.scala

```diff
@@ -0,0 +1,87 @@
+package org.zalando.spearheads.innkeeper
+
+import akka.http.scaladsl.model.{StatusCodes, StatusCode}
+import akka.http.scaladsl.server.Rejection
+
+/**
+ * @author dpersa
+ */
+sealed trait InnkeeperRejection {
+
+  /**
+   * @return the request method followed by the request path
+   */
+  def requestDescription: String
+  def statusCode: StatusCode
+  def message: String
+  def code: String
+}
+
+object Rejections {
```

Comment:
I'll keep the name for now... I find that code looks more readable like this... maybe I'll change the name in the future :)